canonical-web-and-design/maas-ui
# feat(ui): build custom images table

## Done

- Built Custom Images table
- Added selectors for the rest of the tables, and fixed the selectors used for Ubuntu images and Other images (they're only supposed to be synced resources)

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Connect to labmaas and go to /MAAS/r/images
- Check that there is a section for Custom images that lists the data correctly

## Fixes

Fixes canonical-web-and-design/app-squad#83

## Screenshot

![2021-06-29_14-42](https://user-images.githubusercontent.com/25733845/123741590-aca0cc00-d8ed-11eb-8724-8f28a35a1ee1.png)
